src/containers/SignUp/SignUp.js

```diff
@@ -46,13 +46,25 @@ export class SignUp extends Component {
 
   render() {
     return (
-      <form onSubmit={this.handleSubmit}>
-        <input name='name' value={this.state.name} onChange={this.handleChange} />
-        <input name='username' value={this.state.username} onChange={this.handleChange} />
-        <input name='password' value={this.state.password} onChange={this.handleChange} />
-        <button>Submit</button>
-        <span>{this.state.error}</span>
-      </form>
+      <div className='signup-form-div'>
+        <form className='signup-form' onSubmit={this.handleSubmit}>
+          <div className='signup-inner-div'>
+            <label className='signup-label' for='signup-name'>Name</label>
+            <input id='signup-name' className='signup-input' name='name' value={this.state.name} onChange={this.handleChange} />
+          </div>
+          <div className='signup-inner-div'>
+            <label className='signup-label' for='signup-email'>Email</label>
+            <input id='signup-email' className='signup-input' name='username' value={this.state.username} onChange={this.handleChange} />
+          </div>
+          <div className='signup-inner-div'>
+            <label className='signup-label' for='signup-password'>Email</label>
+            <input id='signup-password' className='signup-input' value={this.state.password} onChange={this.handleChange} />
```

Comment:
password wasn't letting me enter text, I think it's because it's missing a name attribute

Reply:
Yes! that is the problem. Let me push up an update!